packages/api-client/src/main/user/UserAPI.ts

```diff
@@ -254,7 +256,28 @@ export class UserAPI {
    * @param parameters Multiple user's handles or IDs
    * @see https://staging-nginz-https.zinfra.io/swagger-ui/#!/users/users
    */
-  public getUsers(parameters: {handles?: string[]; ids?: string[]}): Promise<User[]> {
+  public async getUsers(
+    parameters: {handles?: string[]; ids?: string[]},
```

Comment:
```suggestion
    parameters: {handles?: string[]; ids?: string[]} = {},
```

Reply:
It's exclusive so it is not optional

Reply:
Then we should mark this as exclusive with function overloading.